docs/source/guides/libraries.rst

```diff
@@ -19,17 +19,150 @@ Troubleshooting
 Java Library
 ------------
 
-Where to Get
-^^^^^^^^^^^^
+Prerequisites
+^^^^^^^^^^^^^
 
-How to Use/Import
-^^^^^^^^^^^^^^^^^
+- Java 6
+- Gradle
 
-Example Code
+Build Process
+^^^^^^^^^^^^^
+
+Clone Iroha repository
+
+.. code-block:: shell
+
+  git clone https://github.com/hyperledger/iroha.git --depth=1
+  cd iroha
+
+.. note:: For the latest version checkout a ``develop`` branch by adding 
+  ``-b develop`` parameter.
+
+Now we need to build our Java native library. Run 
+``example/java/build_library.sh``, which will take care of it.
+
+.. code-block:: shell
+
+  cd example/java
+  ./build_library.sh
+
+.. note:: ``build_library.sh`` script creates a ``dist`` folder with
```

Comment:
why dist by the way?

Reply:
Short form of `distribution`. What is more suitable for it? @neewy